config/prod/prow/jobs/custom/test-infra.yaml

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Add a fake plank image here so that autobump.sh can bump only job configs
```

Comment:
It looks this comment is not at the right place. Also it looks you forgot to add the "fake plank image" in the Prow job config?

Reply:
Based on the regex, as long as it's in this file it should be fine: https://github.com/knative/test-infra/blob/4440f3f3adad78518dc68411069b0e495af538f3/config/prod/prow/jobs/custom/test-infra.yaml#L302

Reply:
I see, this is hacky...